scripts/re-id/baseline/test.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function, division
+
+import mxnet as mx
+import numpy as np
+from mxnet import gluon, image, nd
+from mxnet.gluon import nn
+from mxnet.gluon.data.vision import transforms
+
+from networks import resnet18, resnet34, resnet50
+from process import ImageTxtDataset
+
+import time, os, sys
+import scipy.io as sio
+from os import path as osp
+
+
+# Evaluate
+def evaluate(qf,ql,qc,gf,gl,gc):
+    query = qf
+    score = np.dot(gf,query)
+    # predict index
+    index = np.argsort(score)  #from small to large
+    index = index[::-1]
+    #index = index[0:2000]
+    # good index
+    query_index = np.argwhere(gl==ql)
+    camera_index = np.argwhere(gc==qc)
+
+    good_index = np.setdiff1d(query_index, camera_index, assume_unique=True)
+    junk_index1 = np.argwhere(gl==-1)
+    junk_index2 = np.intersect1d(query_index, camera_index)
+    junk_index = np.append(junk_index2, junk_index1) #.flatten())
+    
+    CMC_tmp = compute_mAP(index, good_index, junk_index)
+    return CMC_tmp
+
+
+def compute_mAP(index, good_index, junk_index):
```

Comment:
It's good to have one under https://github.com/dmlc/gluon-cv/tree/master/gluoncv/utils/metrics